plugins/pci-mei/README.md

```diff
@@ -6,3 +6,7 @@ Introduction
 
 This plugin checks if the ME is in Manufacturing Mode. The result will be stored
 in an security attribute for HSI.
+
+External interface access
+---
+????
```

Comment:
Config space of random PCI devices

Reply:
But how does it reach out?  Is it some ioctl to do it?

Reply:
na, it's literally the `config` file which is exported by the kernel in sysfs -- we just read it like a normal file

Reply:
OK, I think in general I want to write specifics that things need, IE `/sys/class/foo/bar/config` as that's much easier for folks to consume.

Reply:
e.g. `/sys/class/pci_bus/0000:01/device/config`